File: Информатика для робототехники/Алтай/9 класс/Вариант 1/A3.cpp
#include <iostream>
#include <vector>

using namespace std;

int main() {
    int n, m;
    cin >> n >> m;
    vector<vector<int>> a(n, vector<int>(m));
    for (int i = 0; i < n; i++) {
        for (int j = 0; j < m; j++) {
            cin >> a[i][j];
        }
    }
    vector<int> w;
    int maxR = 0, r;
    int count;
    for (int i = 0; i < n; i++) {
        r = 0;
        for (int j = 0; j < m; j++) {
            if (a[i][j] > r) r = a[i][j];
        }
        if (r > maxR) {
            w.clear();
            w.push_back(i);
            count = 1;
            maxR = r;
        } else if (r == maxR) {
            w.push_back(i);
            count++;
        }
    }
    cout << count << endl;
    for (auto x : w) {
        cout << x << " ";
    }
    return 0;
}

File: Информатика для робототехники/Алтай/11 класс/Вариант 2/A3.py
w = 'ЗЕРКАЛО'
a = set()
for c1 in w:
    for c2 in w:
        for c3 in w:
            for c4 in w:
                for c5 in w:
                    for c6 in w:
                        s = c1 + c2 + c3 + c4 + c5 + c6
                        if s.count('К') > 4 or s.count('З') > 1 or s.count('Е') > 1 or s.count('Р') > 1 or s.count('А') > 1 or s.count('Л') > 1 or s.count('О') > 1:
                            continue
                        a.add(s)
print(len(a))
for x in a:
    print(x)
